pkg/policy/portmap_test.go

```diff
@@ -56,7 +56,7 @@ func (ds *PolicyTestSuite) TestPolicyNewPortProto(c *C) {
 
 	_, err = newPortProto(80, "cccp")
 	c.Assert(err, Not(IsNil))
-	c.Assert(err.Error(), Equals, "unknown protocol 'cccp'")
+	c.Assert(err.Error(), Equals, "unknown protocol \"cccp\"")
```

Comment:
I'm not sure how this is related to the PR, but it explains why you pulled in @cilium/policy for review. I'd suggest queuing up changes like this in a separate PR in future just to reduce the number of different codeowner groups the PR requires to review.